gen.py

```diff
@@ -156,6 +156,7 @@ def generate_deb(options_param, channel_val):
     if DIST in RPM:
         with open(SPEC_IN, 'r', encoding='utf-8') as SPEC_TMPL:
             for channel in CHANNELS:
+                SPEC_TMPL.seek(0)
```

Comment:
This is a pretty janky way to do this. What you really mean is
```suggestion
        for channel in CHANNELS:
            with open(SPEC_IN, 'r', encoding='utf-8') as SPEC_TMPL:
```
i.e. for each channel, generate the spec from the template.